docs/events/timeline-ui-overview.asciidoc

```diff
@@ -2,84 +2,72 @@
-within a Timeline itself. Additionally, you can add a query directly in Timeline
-(click *+ Add field*).
+other Timelines. Additionally, you can add a query directly in Timeline
+by clicking *+ Add field*.
 
 [role="screenshot"]
 image::images/timeline-ui-updated.png[]
 
 Timelines are responsive, and they persist as you move through the {es-sec-app}
 collecting data. Auto-saving ensures that the results of your investigation are
-available for review by other analysts and incident response teams. To record
-and share your findings with others, attach your Timeline to a
-<<cases-overview, case>>.
+available for later review. To record and share your findings with others, attach your Timeline to a <<cases-overview, case>>.
 
-TIP: An untitled Timeline is saved as a draft. To attach a Timeline to a
+TIP: Untitled Timelines are saved as drafts. To attach a Timeline to a
 <<cases-overview, case>>, you must give it a title.
 
 In addition to Timelines, you can create and attach Timeline templates to
 <<detection-engine-overview, detection rules>>. Timeline templates allow you to
-define the source event fields used when you investigate a detections alert in
+define the source event fields used when you investigate alerts in
 Timeline. You can select whether the fields use predefined values or values
-retrieved from the alert. For more information, see <<timeline-templates-ui>>.
+retrieved from the alert. For more information, refer to <<timeline-templates-ui>>.
 
 
 [discrete]
 [[refine-timeline-results]]
 == View and refine Timeline results
 
-Using the drop-down options
-by the KQL bar, you can select whether <<det-engine-terminology, alerts>>,
-other raw events, or both are displayed in the Timeline.
+Using the drop-down options by the KQL bar, you can select whether <<det-engine-terminology, alerts>>, other raw events, or both are displayed in the Timeline.
 
 [discrete]
 [[conf-timeline-display]]
 == Configure Timeline event context and display
 
-Many events in Timeline are presented in easy-to-follow rendered views that
-include relevant contextual information. These views are called
-*Event Renderers*, and they can be configured via the Settings icon in the upper
-left corner of the result's pane:
+Many types of events automatically appear in preconfigured views that provide various kinds of context, called *Event Renderers*. You can see and turn them on or off via the Settings icon in the upper left corner of the results pane:
 
 [role="screenshot"]
 image::images/timeline-ui-renderer.png[]
 
-The example above displays a flow event renderer. If you see a particular item
-that interests you, you can drag it to the dropzone for further investigation.
+The example above displays the Flow event renderer, which highlights the movement of data between its source and destination. If you see a particular part of the rendered event that interests you, you can drag it up to the dropzone for further investigation.
```

Comment:
Is it "dropzone" or "drop area"? Both are used on this page. (And if "zone," i think two words would be better: "drop zone")

Also, where exactly is this in the UI? You can probably establish this without having to create a new image or GIF; I think you could just say something like "the drop zone [directly?] below the query bar".